src/main/java/com/igorbochkor/dao/impl/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,122 @@
+package com.igorbochkor.dao.impl;
+
+import com.igorbochkor.dao.ManufacturerDao;
+import com.igorbochkor.exception.DataProcessingException;
+import com.igorbochkor.lib.Dao;
+import com.igorbochkor.model.Manufacturer;
+import com.igorbochkor.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String createQuery = "INSERT INTO manufactures (name, country) VALUE (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement
+                    = connection.prepareStatement(createQuery, Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+            statement.close();
+            resultSet.close();
```

Comment:
it is optional to close ResultSet before Statement, because when a Statement object is closed, its current ResultSet object, if one exists, is also closed.
https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/java/sql/Statement.html#close()